submissions/acerpo/index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="ru">
```

Comment:
The whole page uses the English language. but you set russuan language, set the correct value for lang attribute? ita' important for SEO and accessibilty